Workbooks/WorkbooksMetadata.json

```diff
@@ -944,5 +944,18 @@
     "templateRelativePath": "NormalizedNetworkEvents.json",
     "subtitle": "",
     "provider": "Microsoft"
+  },
+     {
+  "workbookKey": "WorkspaceAuditingWorkbook",
+  "logoFileName": "paloalto_logo.svg",
+  "description": "Workspace auditing report\r\nUse this report to understand query runs across your workspace.",
+  "dataTypesDependencies": [ "LAQueryLogs" ],
+  "dataConnectorsDependencies": [ "" ],
+  "previewImagesFileNames": [ "WorkspaceAuditgingWhite.png", "WorkspaceAuditingBlack.png" ],
+  "version": "1.0",
+  "title": "Workspace audit",
```

Comment:
Maybe "query auditing"? Or it is indeed audit on the workspace?

Reply:
It's more than just query auditing so would rather leave the name as is